lib/src/deriv_chart/chart/data_visualization/markers/marker_icon_painters/tick_marker_icon_painter.dart

```diff
@@ -62,14 +67,25 @@ class TickMarkerIconPainter extends MarkerGroupIconPainter {
     EpochToX epochToX,
     QuoteToY quoteToY,
     PainterProps painterProps,
+    AnimationInfo animationInfo,
   ) {
     final Map<MarkerType, Offset> points = <MarkerType, Offset>{};
 
     for (final ChartMarker marker in markerGroup.markers) {
-      final Offset center = Offset(
-        epochToX(marker.epoch),
-        quoteToY(marker.quote),
-      );
+      final Offset center;
+
+      // Special handling for contractMarker - always position on left side
+      if (marker.markerType == MarkerType.contractMarker) {
+        center = Offset(
+          20, // Fixed left position (20 pixels from left edge)
```

Comment:
```suggestion
          20, // Fixed left position (20 pixels from left edge)
```
Can we move this to `markerStyles` instead?